.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to cloudone-filestorage-plugins
+
+:tada: Thank you for taking the time to contribute! :tada:
+
+## How Can I Contribute
+
+- Fork this repository
+- Commit your code with message following [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/).
```

Comment:
Commit your code with message following -> Commit your code with a message that is structured according to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) specification.